### PR TITLE
Merge 4075/4083 commits from OMMR

### PR DESCRIPTION
**Pull request by smlindauer**

### CODE REVIEW

File: src/sasctl/pzmm/pickleModel.py [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-import os
+from pathlib import Path
 
 import pickle
 
 # %%
 class PickleModel():
     
-    def pickleTrainedModel(self, trainedModel, modelPrefix, pPath=os.getcwd()):
+    def pickleTrainedModel(trainedModel, modelPrefix, pPath=Path.cwd()):
```
Comment (jlwalke2) [inline src/sasctl/pzmm/pickleModel.py:13]: Change to
    def pickleTrainedModel(self, trainedModel, modelPrefix, pPath=Path.cwd()):

File: src/sasctl/pzmm/zipModel.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 class ZipModel():
     
-    def zipFiles(self, fileDir, modelPrefix):
+    def zipFiles(fileDir, modelPrefix):
```
Comment (jlwalke2) [inline src/sasctl/pzmm/zipModel.py:12]: Change to 
    def zipFiles(self, fileDir, modelPrefix):

**Review by jlwalke2 — CHANGES_REQUESTED** [review]
Just need to re-add the two missing "self" parameters.

**Comment by smlindauer** [comment]
> Just need to re-add the two missing "self" parameters.

819a2e6 & d8db960 fix the missing self parameters.

**Review by jlwalke2 — APPROVED** [review]

**Review by jlwalke2 — APPROVED** [review]

**Pull request closed without merging**